addons/block_code/ui/block_canvas/block_canvas.gd

```diff
@@ -38,6 +38,8 @@ var zoom: float:
 	get:
 		return _window.scale.x
 
+var _undo_redo: EditorUndoRedoManager
```

Comment:
Personally I would just pass this down in to `build_tree` rather than stashing it in the object. It's only called internally to this object, so there's no danger of it breaking another user.

Reply:
Yes I though about that, I did it like this because is also used in the `build_tree()` method that's called recursively. But yes it can be passed around to `build_tree()` as parameter.